data/help.txt

```diff
@@ -65,9 +68,26 @@ help "lost 7"
 help "map"
 	`This map shows all the star systems you know about and the hyperspace links between them. You can use the map to plot a hyperspace course by clicking on the system you want to travel to. To initiate a jump to that system, close the map and press <Initiate hyperspace jump>.`
 
+help "map advanced"
+	`As you explore more of the galaxy, the star map will quickly become a very powerful tool. Clicking on a distant system will automatically plot the quickest route to it. If you have missions spread across several systems, you can shift-click each to plot a route between them, or control-click a system to inspect missions there without setting a route.`
+	`There are also many other useful modes the map can be set to. Click on one of the buttons in the lower right corner to learn more.`
+
+help "map advanced ports"
+	`The Ports panel provides detailed information about every planet, station, or other stellar objects you've come across. Most information can be clicked on (like the word "Outfitter", or on the price of a trade good), which will color every system on the star map appropriately.`
+	`For instance, you can click on "(has been visited)" to see which planets you've visited or not, or click on a trade good price to show how prices change throughout the galaxy. If you have a route planned already, this can be combined with a control-click to inspect trade good prices at stops along the way. A key will be shown on the right to let you know what colors mean in any given mode.`
+
+help "map advanced shops"
+	`Both the Shipyard and Outfitter panels give the same information: A catalog of any items you've seen in your career, providing detailed statistics at any time. Items will be in categories on the left side of the screen, which can be collapsed individually, or all at once with shift-click on a category name. You can also use the "Find" button in the bottom right to search by name.`
+	`When selecting an item, the details will appear in the top-right. Ports which sell it will be colored yellow, while ports with the correct shop but not the item will be colored teal. You can compare what you've selected with a second item with shift-click, which will put both details side-by-side.`
+
 help "multiple ships"
 	`Now that you have more than one ship, while landed on a planet you can reorder the ships in your fleet by clicking and dragging them in the shipyard, outfitter, or player info panel. You will use the first ship in the list as your flagship. In the player info panel you can also "park" a ship on a planet if you want to travel somewhere without it coming with you.`
```

Comment:
Give the key for the player info panel the first time it is mentioned, which in this case would mean mentioning the key in "multiple ships" instead of "multiple ship controls".
```suggestion
	`Now that you have more than one ship, while landed on a planet you can reorder the ships in your fleet by clicking and dragging them in the shipyard, outfitter, or player info panel. You will use the first ship in the list as your flagship. In the player info panel (accessed with <View player info>) you can also "park" a ship on a planet if you want to travel somewhere without it coming with you.`
```